.github/workflows/publish-setup.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish setup
+
+# Events that trigger this workflow
```

Comment:
Comment not necessary needed.

Reply:
Well, then we need to remove this comment from every GitHub Actions workflow in this and other projects. :see_no_evil:

Reply:
In my opinion, we should at least start at some point. But it is up to you.